src/rules/at-rule-no-unknown/index.js

```diff
@@ -11,6 +11,7 @@ const sassAtRules = [
   "else if",
   "error",
   "extend",
+  "font-face",
```

Comment:
`@font-face` is a CSS at-rule, not Sass, and [stylelint already accounts for it](https://github.com/stylelint/stylelint/blob/19f987e5bb3bf62d8ce634bd8dfe8482d5d76d4a/lib/reference/keywordSets.js#L537).

Reply:
Yeah, I don't think that it should be part of this PR. :)